### PR TITLE
K8SPXC-1082: fix service labels and annotations behaviour

### PR DESCRIPTION
**Pull request by pooknull**
[K8SPXC-1082](https://jira.percona.com/browse/K8SPXC-1082) [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=percona&utm_campaign=badge_info)

https://jira.percona.com/browse/K8SPXC-1082

If there are no annotations or labels in the CR, operator shouldn't overwrite new labels or annotations added to the service

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| affinity-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/affinity-8-0.log) |
| auto-tuning-8-0 | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/auto-tuning-8-0.log) |
| cross-site-8-0 | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/cross-site-8-0.log) |
| demand-backup-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/demand-backup-8-0.log) |
| demand-backup-cloud-8-0 | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/demand-backup-cloud-8-0.log) |
| demand-backup-encrypted-with-tls-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/demand-backup-encrypted-with-tls-8-0.log) |
| haproxy-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/haproxy-5-7.log) |
| haproxy-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/haproxy-8-0.log) |
| init-deploy-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/init-deploy-5-7.log) |
| init-deploy-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/init-deploy-8-0.log) |
| limits-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/limits-8-0.log) |
| monitoring-2-0-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/monitoring-2-0-8-0.log) |
| one-pod-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/one-pod-5-7.log) |
| one-pod-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/one-pod-8-0.log) |
| pitr-8-0 | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/pitr-8-0.log) |
| proxy-protocol-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/proxy-protocol-8-0.log) |
| proxysql-sidecar-res-limits-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/proxysql-sidecar-res-limits-8-0.log) |
| recreate-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/recreate-8-0.log) |
| restore-to-encrypted-cluster-8-0 | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/restore-to-encrypted-cluster-8-0.log) |
| scaling-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/scaling-8-0.log) |
| scaling-proxysql-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/scaling-proxysql-8-0.log) |
| scheduled-backup-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/scheduled-backup-5-7.log) |
| scheduled-backup-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/scheduled-backup-8-0.log) |
| security-context-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/security-context-8-0.log) |
| smart-update1-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/smart-update1-8-0.log) |
| smart-update2-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/smart-update2-8-0.log) |
| storage-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/storage-8-0.log) |
| tls-issue-cert-manager-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/tls-issue-cert-manager-8-0.log) |
| tls-issue-cert-manager-ref-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/tls-issue-cert-manager-ref-8-0.log) |
| tls-issue-self-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/tls-issue-self-8-0.log) |
| upgrade-consistency-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/upgrade-consistency-8-0.log) |
| upgrade-haproxy-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/upgrade-haproxy-8-0.log) |
| upgrade-proxysql-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/upgrade-proxysql-8-0.log) |
| users-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/users-5-7.log) |
| users-8-0 | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/users-8-0.log) |
| validation-hook-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/validation-hook-8-0.log) |
| We run 36 out of 36|

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/1270/commits/6eaaa06ebed009c119555777c6d7eabb8bb732bf
image: `perconalab/percona-xtradb-cluster-operator:PR-1270-6eaaa06e`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| affinity-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/affinity-8-0.log) |
| auto-tuning-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/auto-tuning-8-0.log) |
| cross-site-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/cross-site-8-0.log) |
| demand-backup-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/demand-backup-8-0.log) |
| demand-backup-cloud-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/demand-backup-cloud-8-0.log) |
| demand-backup-encrypted-with-tls-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/demand-backup-encrypted-with-tls-8-0.log) |
| haproxy-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/haproxy-5-7.log) |
| haproxy-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/haproxy-8-0.log) |
| init-deploy-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/init-deploy-5-7.log) |
| init-deploy-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/init-deploy-8-0.log) |
| limits-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/limits-8-0.log) |
| monitoring-2-0-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/monitoring-2-0-8-0.log) |
| one-pod-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/one-pod-5-7.log) |
| one-pod-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/one-pod-8-0.log) |
| pitr-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/pitr-8-0.log) |
| proxy-protocol-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/proxy-protocol-8-0.log) |
| proxysql-sidecar-res-limits-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/proxysql-sidecar-res-limits-8-0.log) |
| recreate-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/recreate-8-0.log) |
| restore-to-encrypted-cluster-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/restore-to-encrypted-cluster-8-0.log) |
| scaling-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/scaling-8-0.log) |
| scaling-proxysql-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/scaling-proxysql-8-0.log) |
| scheduled-backup-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/scheduled-backup-5-7.log) |
| scheduled-backup-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/scheduled-backup-8-0.log) |
| security-context-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/security-context-8-0.log) |
| smart-update1-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/smart-update1-8-0.log) |
| smart-update2-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/smart-update2-8-0.log) |
| storage-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/storage-8-0.log) |
| tls-issue-cert-manager-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/tls-issue-cert-manager-8-0.log) |
| tls-issue-cert-manager-ref-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/tls-issue-cert-manager-ref-8-0.log) |
| tls-issue-self-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/tls-issue-self-8-0.log) |
| upgrade-consistency-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/upgrade-consistency-8-0.log) |
| upgrade-haproxy-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/upgrade-haproxy-8-0.log) |
| upgrade-proxysql-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/upgrade-proxysql-8-0.log) |
| users-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/users-5-7.log) |
| users-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/users-8-0.log) |
| validation-hook-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/6eaaa06e/validation-hook-8-0.log) |
| We run 36 out of 36|

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/1270/commits/6eaaa06ebed009c119555777c6d7eabb8bb732bf
image: `perconalab/percona-xtradb-cluster-operator:PR-1270-6eaaa06e`

File: pkg/controller/pxc/replication.go [hunk 1]
```diff
@@ -55,7 +55,7 @@ func (r *ReconcilePerconaXtraDBCluster) ensurePxcPodServices(cr *api.PerconaXtra
 			return errors.Wrap(err, "failed to set owner to external service")
 		}
 
-		err = r.createOrUpdate(cr, svc)
+		err = r.createOrUpdateService(cr, svc, len(cr.Spec.PXC.Expose.Annotations) == 0)
```
Comment (nmarukovich) [inline pkg/controller/pxc/replication.go:58]: Don't delete annotations. 
STR:
Add annotations to ignore
Add annotation to service (the same behavior for haproxy/proxysql) -> OK
   ```
 serviceAnnotations:
      service.beta.kubernetes.io/aws-load-balancer-backend-protocol: http
```
Only annotations from ignore + annotations from service! 
Than I try to delete service annotations. And see that they were not deleted. 
```
Annotations:              kubernetes.civo.com/cluster-id: 5b4600cd-34c1-4ff5-88e8-4f6ed4198e5e
                          kubernetes.civo.com/firewall-id: ccd93a19-3428-4055-8796-bdd13cfa0c35
                          kubernetes.civo.com/loadbalancer-algorithm: round_robin
                          kubernetes.civo.com/loadbalancer-id: d688ee14-b0ed-45a9-94fe-66c8660c539a
                          kubernetes.civo.com/loadbalancer-name: laboriosam_percona_cluster-pxc-cluster1-proxysql
                          percona.com/last-config-hash:
                            eyJwb3J0cyI6W3sibmFtZSI6Im15c3FsIiwicG9ydCI6MzMwNiwidGFyZ2V0UG9ydCI6MH0seyJuYW1lIjoibXlzcWwtYWRtaW4iLCJwb3J0IjozMzA2MiwidGFyZ2V0UG9ydCI6MH...
                          service.beta.kubernetes.io/aws-load-balancer-backend-protocol: http
```
Logic for annotations should be also implement on labels.

**Review by nmarukovich — CHANGES_REQUESTED** [review]

File: pkg/controller/pxc/replication.go (hunk 1 above)
Comment (hors) [reply pkg/controller/pxc/replication.go:58]: @nmarukovich it is by design

**Review by inelpandzic — APPROVED** [review]

**Review by egegunes — APPROVED** [review]

**Review by hors — APPROVED** [review]

**Review by nmarukovich — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| affinity-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/affinity-8-0.log) |
| auto-tuning-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/auto-tuning-8-0.log) |
| cross-site-8-0 | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/cross-site-8-0.log) |
| demand-backup-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/demand-backup-8-0.log) |
| demand-backup-cloud-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/demand-backup-cloud-8-0.log) |
| demand-backup-encrypted-with-tls-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/demand-backup-encrypted-with-tls-8-0.log) |
| haproxy-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/haproxy-5-7.log) |
| haproxy-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/haproxy-8-0.log) |
| init-deploy-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/init-deploy-5-7.log) |
| init-deploy-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/init-deploy-8-0.log) |
| limits-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/limits-8-0.log) |
| monitoring-2-0-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/monitoring-2-0-8-0.log) |
| one-pod-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/one-pod-5-7.log) |
| one-pod-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/one-pod-8-0.log) |
| pitr-8-0 | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/pitr-8-0.log) |
| proxy-protocol-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/proxy-protocol-8-0.log) |
| proxysql-sidecar-res-limits-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/proxysql-sidecar-res-limits-8-0.log) |
| recreate-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/recreate-8-0.log) |
| restore-to-encrypted-cluster-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/restore-to-encrypted-cluster-8-0.log) |
| scaling-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/scaling-8-0.log) |
| scaling-proxysql-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/scaling-proxysql-8-0.log) |
| scheduled-backup-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/scheduled-backup-5-7.log) |
| scheduled-backup-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/scheduled-backup-8-0.log) |
| security-context-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/security-context-8-0.log) |
| smart-update1-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/smart-update1-8-0.log) |
| smart-update2-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/smart-update2-8-0.log) |
| storage-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/storage-8-0.log) |
| tls-issue-cert-manager-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/tls-issue-cert-manager-8-0.log) |
| tls-issue-cert-manager-ref-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/tls-issue-cert-manager-ref-8-0.log) |
| tls-issue-self-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/tls-issue-self-8-0.log) |
| upgrade-consistency-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/upgrade-consistency-8-0.log) |
| upgrade-haproxy-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/upgrade-haproxy-8-0.log) |
| upgrade-proxysql-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/upgrade-proxysql-8-0.log) |
| users-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/users-5-7.log) |
| users-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/users-8-0.log) |
| validation-hook-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/validation-hook-8-0.log) |
| We run 36 out of 36|

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/1270/commits/8cda99f79efccebefe6e2b9bfe0da2f2da74728b
image: `perconalab/percona-xtradb-cluster-operator:PR-1270-8cda99f7`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| affinity-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/affinity-8-0.log) |
| auto-tuning-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/auto-tuning-8-0.log) |
| cross-site-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/cross-site-8-0.log) |
| demand-backup-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/demand-backup-8-0.log) |
| demand-backup-cloud-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/demand-backup-cloud-8-0.log) |
| demand-backup-encrypted-with-tls-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/demand-backup-encrypted-with-tls-8-0.log) |
| haproxy-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/haproxy-5-7.log) |
| haproxy-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/haproxy-8-0.log) |
| init-deploy-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/init-deploy-5-7.log) |
| init-deploy-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/init-deploy-8-0.log) |
| limits-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/limits-8-0.log) |
| monitoring-2-0-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/monitoring-2-0-8-0.log) |
| one-pod-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/one-pod-5-7.log) |
| one-pod-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/one-pod-8-0.log) |
| pitr-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/pitr-8-0.log) |
| proxy-protocol-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/proxy-protocol-8-0.log) |
| proxysql-sidecar-res-limits-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/proxysql-sidecar-res-limits-8-0.log) |
| recreate-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/recreate-8-0.log) |
| restore-to-encrypted-cluster-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/restore-to-encrypted-cluster-8-0.log) |
| scaling-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/scaling-8-0.log) |
| scaling-proxysql-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/scaling-proxysql-8-0.log) |
| scheduled-backup-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/scheduled-backup-5-7.log) |
| scheduled-backup-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/scheduled-backup-8-0.log) |
| security-context-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/security-context-8-0.log) |
| smart-update1-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/smart-update1-8-0.log) |
| smart-update2-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/smart-update2-8-0.log) |
| storage-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/storage-8-0.log) |
| tls-issue-cert-manager-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/tls-issue-cert-manager-8-0.log) |
| tls-issue-cert-manager-ref-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/tls-issue-cert-manager-ref-8-0.log) |
| tls-issue-self-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/tls-issue-self-8-0.log) |
| upgrade-consistency-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/upgrade-consistency-8-0.log) |
| upgrade-haproxy-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/upgrade-haproxy-8-0.log) |
| upgrade-proxysql-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/upgrade-proxysql-8-0.log) |
| users-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/users-5-7.log) |
| users-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/users-8-0.log) |
| validation-hook-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1270/8cda99f7/validation-hook-8-0.log) |
| We run 36 out of 36|

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/1270/commits/8cda99f79efccebefe6e2b9bfe0da2f2da74728b
image: `perconalab/percona-xtradb-cluster-operator:PR-1270-8cda99f7`

**Pull request merged**